components/sup/src/util/pkg.rs

```diff
@@ -52,7 +53,7 @@ where
         // function, though, should always have a channel. We should
         // push this "Option-ness" as far down the stack as we can,
         // with the ultimate goal of eliminating it altogether.
-        Some(channel),
+        Some(&channel),
```

Comment:
🤔 How far out are we from getting rid of this `Option` and having everything take a channel?

cc: @chefsalim

Reply:
I'll leave that one to @chefsalim